src/util/cleaners/asBase64.js

```diff
@@ -4,7 +4,7 @@ import { asCodec, asString, uncleaner } from 'cleaners'
 import { base64 } from 'rfc4648'
 
 export const asBase64 = asCodec(
-  raw => base64.parse(asString(raw)),
+  raw => base64.parse(asString(raw), { loose: true }),
```

Comment:
Do not put `loose` in here! It is going to mask other errors like this in the future.

Reply:
Woops, that creeped in from the _original, incorrect fix_. Thanks for catching this; sorry for being sloppy.